live-ingester/pom.xml

```diff
@@ -59,6 +49,17 @@
 			<artifactId>acs-event-model</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>software.amazon.awssdk</groupId>
+			<artifactId>s3</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.github.tomakehurst</groupId>
```

Comment:
Should this be `org.wiremock`? https://wiremock.org/docs/download-and-installation/

Reply:
Good observation! I was looking at: https://mvnrepository.com/search?q=wiremock and some I become careless 🤦‍♂️ Already fixed, thanks!